web-test-runner.config.mjs

```diff
@@ -0,0 +1,16 @@
+export default {
```

Comment:
Please, remove this file. There's no need for these file to be loaded in this context. They are only used with the authorization element and Digest method. The command `"test": "web-test-runner test/**/*.test.js --coverage --node-resolve --playwright --browsers chromium firefox webkit",` is enough.